### ISSUE TITLE
[clang-format] QualifierAlignment does not respect typedef'd Types

### ISSUE THREAD

**Issue by oefirouz** [llvm/llvm-project#58696]
```c
int main() {
    const int a = 5;
    const int64_t b = 3;
    return 0;
}
```

clang-format with `QualifierAlignment: Right` gives

```c
int main() {
    int const a = 5;
    const int64_t b = 3;
    return 0;
}
```

The same issue comes up if I `typdef int myint_t`.

**Comment by oefirouz** [comment]
If someone could help direct me on where/how to fix this I would love to help out as well. However I'm unfamiliar with the clang codebase.

**Comment by rymiel** [comment]
This should be fixed by cd7ab4b5c1684dcc60de027700177adfa096b98c
